src/pages/workspaces/workspace/Workflows.js

```diff
@@ -479,8 +478,55 @@ export const Workflows = _.flow(
       loading && spinnerOverlay,
     ]),
   ]);
+};
+
+export const Workflows = _.flow(
+  forwardRefWithName('Workflows'),
+  wrapWorkspace({
+    breadcrumbs: (props) => breadcrumbs.commonPaths.workspaceDashboard(props),
+    title: 'Workflows',
+    activeTab: 'workflows',
+  })
+)(({ namespace, name, workspace: ws, workspace: { workspace } }, ref) => {
+  if (isAzureWorkspace(ws)) {
+    return h(AzureWorkflows, ws);
+  }
+
+  return h(GcpWorkflows, { namespace, name, ws, workspace, ref });
 });
 
+export const AzureWorkflows = ({ workspace }) => {
+  // State
+  const [cbasStatus, setCbasStatus] = useState();
+  const [cromwellStatus, setCromwellStatus] = useState();
+  const signal = useCancellation();
+
+  const loadCbasStatuses = async () => {
+    try {
+      const workspaceId = workspace.workspaceId;
+
+      const appUrl = (
+        await Apps(signal)
+          .listAppsV2(workspaceId)
+          .then((apps) => resolveRunningCromwellAppUrl(apps, getUser()?.email))
+      ).cbasUrl;
+
+      const status = await Ajax(signal).Cbas.status(appUrl);
+      setCbasStatus(JSON.stringify(status.ok));
+      setCromwellStatus(JSON.stringify(status.systems.cromwell.ok));
+    } catch (error) {
+      notify('error', 'Error loading statuses', { detail: error instanceof Response ? await error.text() : error });
+    }
+  };
+
+  // Lifecycle
+  useOnMount(async () => {
+    await loadCbasStatuses();
```

Comment:
I don't think you need `await` here because there is nothing happening after this call in `useOnMount` that needs `loadCbasStatuses` to have completed.